odoo/tests/common.py

```diff
@@ -311,6 +311,14 @@ class BaseCase(TreeCase, MetaCase('DummyCase', (object,), {})):
     longMessage = True      # more verbose error message by default: https://www.odoo.com/r/Vmh
     warm = True             # False during warm-up phase (see :func:`warmup`)
 
+    def setUp(self):
+        def iap_jsonrpc(url, method='call', params=None, timeout=15):
```

Comment:
```suggestion
        def iap_jsonrpc(*args, **kwargs):
```
The args are not used in the mock.